pkgs/development/python-modules/frozenlist2/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, setuptools
+}:
+
+buildPythonPackage rec {
```

Comment:
using rec is an anti-pattern, ref: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293/18 / https://github.com/NixOS/nixpkgs/pull/119942

Reply:
Yeah I know but https://github.com/NixOS/nixpkgs/pull/271387 is not merged yet

Reply:
ok, thanks